tools/synthesize_test_cases.py

```diff
@@ -433,7 +433,10 @@ def progress_bar_handler(executor_dict, verbose, desc):
         else:
             dev_versions_to_include_list = previous_fim_list
     if config == 'PREV':
-        dev_versions_to_include_list = dev_versions_to_compare
```

Comment:
@RyanSpies-NOAA Is this change something you want to merge in, or is it from testing? Only reason I ask is because this PREV `if` block now matches the DEV `if` block immediately above. If this is something you want to merge, we should probably remove the outer 2 `if config == 'DEV/PREV'` block as it's redundant.

Reply:
Good catch on the redundancy here - I'll fix that now. I've been meaning to ask the group if there were any issues with making this change. If I remember correctly, I added this because you previously would get an error if you ran with `PREV` flag and did not specify any dev compare versions using the`-dc` flag. I set the default to always include the `previous_fim_list` to avoid the error. Do you see any issue with this set up?

Reply:
I don't see any problems with it. It'll just add more versions to the output CSV, right? If so, that doesn't change much besides giving you more data to play with for `eval_plots`.

Reply:
Correct - that was my thought as well.